imagedephi/models/rules_models.py

```diff
@@ -0,0 +1,41 @@
+import importlib.resources
+from typing import Literal
+
+from pydantic import BaseModel
+import yaml
+
+
+class Replace(BaseModel):
+    method: Literal["replace"]
+    new_value: str
+
+
+class Keep(BaseModel):
+    method: Literal["keep"]
+
+
+class Delete(BaseModel):
+    method: Literal["delete"]
+
+
+class Rule(BaseModel):
+    description: str | None = None
```

Comment:
@brianhelba we discussed default `None` vs truly optional. This thread does offer some interesting solutions but I'm not sure if any would work for us.
https://github.com/pydantic/pydantic/issues/1223#issuecomment-589986747